pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         - name: csi-attacher
           image: {{ .AttacherImage }}
           args:
-            - "--v={{ .LogLevel }}"
+            - "--v={{ .SidecarLogLevel }}"
```

Comment:
Would we need granularity for each sidecar log level? Maybe it's not worth it, but at the same time it may not make sense to turn up verbosity on all sidecars if you are only troubleshooting one of them.

Reply:
I think it should be fine to keep it for all sidecars as this will be turned on/off for a short period of debugging. if we add per sidecar, it might be more complicated and not worthy as you said.

Reply:
Agreed, let's go with this simple setting for now.